examples/06_assets/main.rs

```diff
@@ -1,66 +1,32 @@
 extern crate amethyst;
 extern crate cgmath;
-extern crate obj;
 
-use amethyst::engine::{Application, State, Trans};
-use amethyst::context::{ContextConfig, Context};
 use amethyst::config::Element;
-use amethyst::ecs::{World, Join};
-use amethyst::context::asset_manager::{Assets, AssetLoader, AssetLoaderRaw, DirectoryStore, Mesh, Texture};
-use amethyst::renderer::VertexPosNormal;
-use cgmath::{InnerSpace, Vector3};
-use std::io::BufReader;
-
-struct Obj(obj::Obj);
```

Comment:
I think the goal of the pull request is excellent, more standard formats supported by the asset manger would be fantastic. I also agree with @Emilgardis that there shoud be an example demonstrating custom asset loading. Maybe do both in 06_assets? Move obj loader code to asset mananger and implement some obviously custom format loader in the example code and load one obj asset and one custom asset?